ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -206,6 +206,7 @@ wan_rr:
           static_routes:
             - prefix: 0.0.0.0/0
           ip_address: 10.7.7.7/31
+          public_ip: 9.8.8.8/24
```

Comment:
please also add a public_ip to cv-pathfinder-pathfinder2 for instance and make sure that when overwritten in the CV_PATHFINDER_MULTI_RR_TESTS group_vars file, the value from under `wan_route_servers` wins over `public_ip`. Also please add a comment above `public_ip` in this file to indicate that this is the expected behavior.